profile/2019-03-01-hybrid.json

```diff
@@ -6,175 +6,472 @@
+                }
             ],
             "2016-03-01": [
-                "availableStacks",
-                "deploymentLocations",
-                "georegions",
-                "ishostnameavailable",
-                "isusernameavailable",
-                "listSitesAssignedToHostName",
-                "publishingUsers",
-                "recommendations",
-                "runtimes",
-                "sourceControls",
-                "validate"
+                {
+                    "resourceName": "availableStacks",
+                    "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/web/resource-manager/Microsoft.Web/stable/2016-03-01/Provider.json"
+                },
+                {
+                    "resourceName": "deploymentLocations",
+                    "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/web/resource-manager/Microsoft.Web/stable/2016-03-01/ResourceProvider.json"
+                },
+                {
+                    "resourceName": "georegions",
+                    "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/web/resource-manager/Microsoft.Web/stable/2016-03-01/ResourceProvider.json"
+                },
+                {
+                    "resourceName": "listSitesAssignedToHostName",
+                    "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/web/resource-manager/Microsoft.Web/stable/2016-03-01/ResourceProvider.json"
+                },
+                {
+                    "resourceName": "publishingUsers",
+                    "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/web/resource-manager/Microsoft.Web/stable/2016-03-01/ResourceProvider.json"
+                },
+                {
+                    "resourceName": "recommendations",
+                    "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/web/resource-manager/Microsoft.Web/stable/2016-03-01/Recommendations.json"
+                },
+                {
+                    "resourceName": "sourceControls",
+                    "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/web/resource-manager/Microsoft.Web/stable/2016-03-01/ResourceProvider.json"
+                },
+                {
+                    "resourceName": "validate",
+                    "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/web/resource-manager/Microsoft.Web/stable/2016-03-01/ResourceProvider.json"
+                }
             ]
         }
     },
     "data-plane": {
         "microsoft.keyvault": {
-            "2016-10-01": []
+            "2016-10-01": [
+                {
+                    "resourcename": "*",
```

Comment:
* should no be here, we should list all the supported resource types explicilty here.. I am not sure what resource type means for data plane